### ISSUE TITLE
Learning rate schedulers code snippet in documentation

### ISSUE THREAD

**Issue by eviatarbach** [CliMA/EnsembleKalmanProcesses.jl#311]
It would be good to have a small code snippet of how to use the learning rate schedulers in [learning_rate_scheduler.md](https://github.com/CliMA/EnsembleKalmanProcesses.jl/blob/main/docs/src/learning_rate_scheduler.md), especially dealing with the termination condition, since this is not obvious now.
